UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs

```diff
@@ -151,17 +150,10 @@ protected override void UpdateSpriteConstructionState()
 
 		private void UpdateSpriteBinState()
 		{
-			if (MachineUnattached)
-			{
-				baseSpriteHandler.ChangeSprite((int) BinSprite.Sideways);
-				overlaysSpriteHandler.PushClear();
-				return;
-			}
-
 			switch (binState)
 			{
 				case BinState.Disconnected:
-					baseSpriteHandler.ChangeSprite((int) BinSprite.Upright);
```

Comment:
Disconnected should be upright. Disconnected refers to the state where you first: turn off the power using the UI, then use a screwdriver to disconnect the power cable as per /tg/.

The sideways spright is for when the disposal bin is unwrenched and moveable.